.github/workflows/fly-deploy-prod.yml

```
@@ -0,0 +1,30 @@
+      - uses: actions/checkout@v4
+      - uses: superfly/flyctl-actions/setup-flyctl@master
+      - run: flyctl deploy -c fly-prod.toml --remote-only --build-only --push --no-cache --image-label "deployment-${{ github.ref_name }}"
+        env:
+          FLY_API_TOKEN: ${{ secrets.FLY_PROD_API_TOKEN }}
+
+  prompt-service:
+    name: Push to Service (Production Environment)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Send POST request
+        run: |
```

actionlint (1.7.4)

got unexpected character '#' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '

(expression)